packages/client/src/pages/Home/components/Rating/rating.module.css

```
@@ -0,0 +1,63 @@
+.containerRating {
+  background-color: rgb(236, 255, 197);
+  border-radius: 40px 40px 0 0;
+  margin-top: -30px;
+  font-size: 32px;
+}
+
+.divider {
+  width: 100%;
+  border: 2px solid rgb(252, 127, 97);
+}
+
+.title {
+  font-size: 100px;
```

Здесь для шрифта можно готовый миксин из `app.css` использовать, там все эти свойства для шрифта есть. Пример можешь глянуть в компоненте `Title` или `TextField`
```suggestion
  composes: font-design-headline-l from global;
```